### PR TITLE
exit button finished without errors i think

### PR DESCRIPTION
**Pull request by leenderteloff**
Added a exit button that checks where the nearest exit is and shows it to the user

### CODE REVIEW

File: MobileDevProject/ViewController.swift [hunk 1]
```diff
@@ -217,7 +222,27 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
...
+                exitImageSouth.tintColor = UIColor.darkText
+                // checks with beacon is nearest and changes the color of the exit nearest to the beacon to red.
+                if nearestBeacon.value == 832 {
+                    exitImageNorth.tintColor = UIColor.red
+                }else if nearestBeacon.value == 748{
+                    exitImageSouth.tintColor = UIColor.red
+                }else if beaconsArray.count == 0{
+                    exitImageNorth.tintColor = UIColor.red          // if there is no beacon found it will change the north exit to red
+                }
+                return
```
Comment (henrycity) [inline MobileDevProject/ViewController.swift]: Indentation!

**Comment by henrycity** [comment]
Check the warning and error from swiftlint before building.

**Review by henrycity — CHANGES_REQUESTED** [review]
Check the linter.

File: MobileDevProject/ViewController.swift [hunk 2]
```diff
@@ -217,7 +222,27 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
...
+                exitImageNorth.tintColor = UIColor.darkText
+                exitImageSouth.image = exitImageSouth.image!.withRenderingMode(.alwaysTemplate)
+                exitImageSouth.tintColor = UIColor.darkText
+                // checks with beacon is nearest and changes the color of the exit nearest to the beacon to red.
+                if nearestBeacon.value == 832 {
+                    exitImageNorth.tintColor = UIColor.red
+                }else if nearestBeacon.value == 748{
+                    exitImageSouth.tintColor = UIColor.red
+                }else if beaconsArray.count == 0{
+                    exitImageNorth.tintColor = UIColor.red          // if there is no beacon found it will change the north exit to red
```
Comment (henrycity) [inline MobileDevProject/ViewController.swift]: Why is the exitImageNorth red if there is no beacon found?

File: MobileDevProject/ViewController.swift [hunk 3]
```diff
@@ -217,7 +222,27 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
...
                 present(actionController, animated: true, completion: nil)
-            default:
+            case exitButton:
+                if nearestBeacon == nil{
+                    exitImageNorth.image = exitImageNorth.image!.withRenderingMode(.alwaysTemplate)
+                    exitImageNorth.tintColor = UIColor.red
+                    return
+                }
+                // changes both exit images to black
+                exitImageNorth.image = exitImageNorth.image!.withRenderingMode(.alwaysTemplate)
```
Comment (henrycity) [inline MobileDevProject/ViewController.swift]: Change to color to gray might be a better idea.

File: MobileDevProject/ViewController.swift (hunk 2 above)
Comment (leenderteloff) [reply MobileDevProject/ViewController.swift]: I'll show both

**Comment by henrycity** [comment]
Don't open a new PR.

**Pull request merged**